README.md

```diff
@@ -2,112 +2,157 @@
 
-### Manage Jira Subscriptions
+![Use issue keys to link your development work to Jira](./docs/images/jira-issue.png)
 
-Admins of an installation can view and delete subscriptions to other Jira instances, without having to log in to the Jira instance itself:
+![See development insights in Jira issues and quickly jump into action.](./docs/images/devinfo.png)
 
-![image](https://user-images.githubusercontent.com/13207348/59897474-d2b83b00-93ba-11e9-81cd-93e0f3db0833.png)
+### See Jira issues in GitHub
+If an issue body contains a valid Jira issue key on your instance, the integration will automatically expand it into a reference link when surrounded in brackets `[]`. For example: [DEV-2095] will be turned into a link to `https://<your-instance>.atlassian.net/browse/DEV-2095`.
 
-This is useful if your installation is setup to send Development information to an instance you no longer have access to, or to audit instances other admins in your org may have previously configured. This only gives you the permission to delete the connection, and will not give you access to the instance itself. You will still need to be granted access in Jira if you want to be able to view the Development information that's been sent to that instance.
+This makes it so Jira issues can be linked inside a comment without it interrupting the flow of the comment as a whole.
 
-## Migrating from the DVCS connector
-Existing users of Jira's built-in DVCS connector that meet the [requirements](#requirements) should migrate to this integration. If you've not yet been prompted to do so, you can manually kick off the migration by:
-1. Sign in to your Jira Cloud account
-2. From the left sidebar in Jira, select Jira Settings -> Applications -> DVCS accounts.
-3. Follow the prompt to upgrade your GitHub connection
+### See GitHub CI/CD data in Jira
+GitHub Actions is a feature from GitHub for automation such as CI/CD. If you’re setting this up for the first time, follow [GitHub Actions Documentation - GitHub Docs](https://docs.github.com/en/actions). If you already have GitHub Actions and want to see CI/CD data from Github in Jira, include the Jira issue key in your commit message, branch name, or PR.
+
+### How the integration works
+When a workflow (e.g. GitHub Action) or development event (e.g. pull request, commit, branch) runs, our app receives a webhook from GitHub. We then extract the issue key from the respective branch/commit/PR and send this information to Jira.
+
+## Migrate from the DVCS Connector
+Existing users of Jira's built-in DVCS connector that meet the [requirements](https://github.com/atlassian/github-for-jira/blob/main/README.md#requirements) should migrate to this integration. If you've not yet been prompted to do so, you can manually kick off the migration by:
```

Comment:
Fairly sure you can change the link to just be `#requirements` so it doesn't reload the page.